File: java_quarkus/src/main/java/com/example/model/Persona.java
package com.example.model;

import io.quarkus.hibernate.orm.panache.PanacheEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.Table;

@Entity
@Table(name = "personas")
public class Persona extends PanacheEntity {
    public String nombre;
    public String apellido;
    public String documento;
    public String direccion;
}

File: kotlin_spring/src/main/kotlin/com/example/kotlin_spring/model/Persona.kt
package com.example.kotlin_spring.model

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "personas")
data class Persona(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0,
    val nombre: String,
    val apellido: String,
    val documento: String,
    val direccion: String
)